README.md

```diff
@@ -1,5 +1,7 @@
 # ColPrac: Contributor's Guide on Collaborative Practices for Community Packages
 
+[![Global Docs](https://img.shields.io/badge/docs-SciML-blue.svg)](https://docs.sciml.ai/ColPrac/stable/)
```

Comment:
this gives me `403 Forbidden`

Reply:
@nickrobinson251 yeah that is correct. We were just discussing this on slack. 

Some of the new docs have security key issues. So Chris suggested using the stable docs still, and we will fix the security key issues. Hopefully this should be fixed soon. I will check for you.

Reply:
Here is an excerpt from the slack conversation.
```
Chris Rackauckas
  [1 hour ago](https://julialang.slack.com/archives/CN04R7WKE/p1666392053485539?thread_ts=1666286927.762689&cid=CN04R7WKE)
and stable might not work on global docs for some because of keys issues, but just point to stable anyways


Krishna Bhogaonker
  [1 hour ago](https://julialang.slack.com/archives/CN04R7WKE/p1666392105818959?thread_ts=1666286927.762689&cid=CN04R7WKE)
Yep, sounds good.


Arno Strouwen
  [1 hour ago](https://julialang.slack.com/archives/CN04R7WKE/p1666392116550959?thread_ts=1666286927.762689&cid=CN04R7WKE)
(but if stable does not exist, make sure the link is correct by checking if /dev/ does exist)
```